packages/image_picker/image_picker_ios/ios/image_picker_ios/Sources/image_picker_ios/FLTPHPickerSaveImageToPathOperation.m

```diff
@@ -128,64 +128,20 @@ - (void)start {
 - (void)processImage:(NSData *)pickerImageData API_AVAILABLE(ios(14)) {
   UIImage *localImage = [[UIImage alloc] initWithData:pickerImageData];
 
-  PHAsset *originalAsset;
-  // Only if requested, fetch the full "PHAsset" metadata, which requires  "Photo Library Usage"
-  // permissions.
-  if (self.requestFullMetadata) {
-    originalAsset = [FLTImagePickerPhotoAssetUtil getAssetFromPHPickerResult:self.result];
```

Comment:
Does anything use `getAssetFromPHPickerResult` now? Can we remove it?

Reply:
Removed function and a test that uses it.